testing/upload_validation.ts

```diff
@@ -1060,16 +1060,34 @@ function validateFormulas(schema: z.ZodObject<any>) {
       {message: 'A formula namespace must be provided whenever formulas are defined.', path: ['formulaNamespace']},
     )
     .refine(
-      metadata =>
-        !(
-          metadata.defaultAuthentication?.type === AuthenticationType.CodaApiHeaderBearerToken &&
-          metadata.networkDomains?.some(
-            (domain: string) => !['coda.io', 'codahosted.io', 'localhost', 'calc-grpc-proxy'].includes(domain),
-          )
-        ),
+      untypedMetadata => {
+        const metadata = untypedMetadata as PackVersionMetadata;
+        if (metadata.defaultAuthentication?.type !== AuthenticationType.CodaApiHeaderBearerToken) {
+          return true;
+        }
+
+        const codaDomains = ['coda.io', 'codahosted.io', 'localhost', 'calc-grpc-proxy'];
```

Comment:
Seems like maybe `codahosted.io` and `calc-grpc-proxy` should be removed from this list of coda domains, since they don't need auth tokens?...

Reply:
Yeah, I don't recall why they made it into this particular validation, @huayang-codaio do you recall?

Reply:
I'm not sure what `localhost` is here for either, but I removed `codahosted.io` and `calc-grpc-proxy` from the list at least because I don't see what either of those domains would do with a coda bearer token

Reply:
I think we were sending creds to `codahosted.io` before the change to always allow the CDN domain. 

`calc-grpc-proxy` and `localhost` were used by the debug pack (which uses the coda api bearer token auth). I am not sure if it's still using these two domains though (maybe not as from some earlier changes). There's a separate check to ensure that these local domains are only used by this pack.

Reply:
The debug pack is already updated so that `coda.io` is its only auth domain